tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -1630,10 +1630,50 @@ def test_list_recommendations_cpu_mem_optimised(cluster_type: str):
                 if INFO_COST_RECOMMENDATIONS_AVAILABLE_CODE in short_term_recommendation["notifications"]:
                     assert "variation" in short_term_recommendation["recommendation_engines"]["cost"]
                     assert "config" in short_term_recommendation["recommendation_engines"]["cost"]
+                    assert short_term_recommendation["recommendation_engines"]["cost"]["config"]["requests"]["cpu"]["amount"] == OPTIMISED_CPU
+                    assert short_term_recommendation["recommendation_engines"]["cost"]["config"]["limits"]["cpu"]["amount"] == OPTIMISED_CPU
+                    assert short_term_recommendation["recommendation_engines"]["cost"]["config"]["requests"]["memory"]["amount"] == OPTIMISED_MEMORY
+                    assert short_term_recommendation["recommendation_engines"]["cost"]["config"]["limits"]["memory"]["amount"] == OPTIMISED_MEMORY
+
+                    if j > 671:
+                        assert "variation" in medium_term_recommendation["recommendation_engines"]["cost"]
```

Comment:
Please move all this as well into a function and pass the term object as a parameter to it.

Reply:
Thanks for the review @chandrams. I have made the changes accordingly.